cdc_rsync/cdc_rsync_client.cc

```diff
@@ -46,8 +46,8 @@ constexpr int kExitCodeNotFound = 127;
 
 constexpr int kForwardPortFirst = 44450;
 constexpr int kForwardPortLast = 44459;
-constexpr char kGgpServerFilename[] = "cdc_rsync_server";
-constexpr char kRemoteToolsBinDir[] = "~/.cache/cdc_file_transfer/";
+constexpr char kCdcServerFilename[] = "cdc_rsync_server";
+constexpr char kRemoteToolsBinDir[] = "~/.cache/cdc-file-transfer/";
```

Comment:
Maybe we should have a `bin` dir in the chache dir?

```suggestion
constexpr char kRemoteToolsBinDir[] = "~/.cache/cdc-file-transfer/bin/";
```

Reply:
Done.